packages/lib/src/components/Card/Card.tsx

```diff
@@ -85,7 +87,12 @@ export class CardElement extends UIElement<CardElementProps> {
                 showPanel
             },
             // installmentOptions of a session should be used before falling back to the merchant configuration
-            installmentOptions: props.session?.configuration?.installmentOptions || props.installmentOptions ,
+            installmentOptions: props.session?.configuration?.installmentOptions || props.installmentOptions,
+            clickToPayConfiguration: {
+                ...props.clickToPayConfiguration,
+                shopperIdentityValue: props.clickToPayConfiguration?.shopperIdentityValue || props?._parentInstance?.options?.session?.shopperEmail,
```

Comment:
If a `shopperEmail` has come from the `/sessions` endpoint do we want the merchant to be able to overwrite it with their manual config?

Reply:
I thought about this when developing - I believe that if the merchant explicitly set the shopperEmail in the configuration object, then it should be overridden. 

Ideally the merchant that is integrating using `sessions` just need to pass the `shopperEmail` to the session creation, and add the `useClickToPay` flag to make it all work - it doesn't make sense to add the `shopperEmail` to the CtP config when using sessions, but if so, then we will use that value